Documentation/contributing/release/backports.rst

```diff
@@ -65,10 +65,11 @@ process for backporting these PRs:
 One-time Setup
 ~~~~~~~~~~~~~~
 
-#. The scripts referred to below need to be run on Linux, they do not
-   work on macOS. You can use the cilium dev VM for this, but you need
-   to configure git to have your name and email address to be used in
-   the commit messages:
+#. The scripts referred to below need to be run on Linux, they do not work on
+   macOS. It is recommend to use the dev VM (``contrib/vagrant/start.sh``), as
```

Comment:
Why is that? We call out the required dependency packages along with their versions, no? 
nit: *recommended

Reply:
A user can spin up their own vagrant VM which has old Python libraries that cause problems with PyGithub for example

Reply:
>We call out the required dependency packages along with their versions

You marked them as required in your 1st commit?

>We might as well point them at a well-maintained VM
We already do, no?

Reply:
Yes, we call out the dependencies, but that doesn't guarantee that the underlying Python libraries on the system for example are compatible with the versions of the dependencies we require.

Reply:
AFAIR, we still need to install couple of packages for the backporting process. Did you try running all the backporting scripts in a dev VM?

Reply:
I've not used the dev VM for backporting because I'm already on Linux. I just want to make it easier for those who do. I believe you use the dev VM for backporting--do you remember what additional packages those are? Can you add them to the list that I've modified. I figured because we already have the dev VM used by many others, it's going to be the most maintained, and since it'll likely be used for development purposes anyway, why not recommend it.